### PR TITLE
Implement allowances and sites pages UX/UI improvements

### PR DESCRIPTION
**Pull request by GabrielSoga**
### Describe the changes you have made in this PR

Budgets UX/UI improvements:

- On popup view of a page that has not have a budget set, there should be a link to do it instead of empty budget bar (right now). Clicking it leads to Edit Preferences modal.
- An icon button of that opens Edit Preferences modal should be in the corner of site box.
- Modal has updated copy
- New disconnect button in Edit Preferences modal.
- New design for the Progress Bar
- Changing "ACTIVE" badge to "BUDGET"
- A revamp of website page views:
  - Logo, name and URL is on the left side
  - Site settings button that opens Edit Preferences modal
  - Displaying info about payments, budget and permissions (badges). If budget not set, showing a link

### Link this PR to an issue

https://github.com/getAlby/lightning-browser-extension/issues/2458

### Type of change

(Remove other not matching type)

- `feat`: New feature (non-breaking change which adds functionality)

### How has this been tested?

Manual testing

### Checklist

- [x] Self-review of changed code
- [x] Manual testing
- [ ] Added automated tests where applicable
- [ ] Update Docs & Guides
- For UI-related changes
- [ ] Darkmode
- [ ] Responsive layout

### CODE REVIEW

**Comment by reneaaron** [comment]
@fczuardi @GabrielSoga Thanks for this PR! :+1: Could you please have a look at the unit tests? Some of them seem to fail currently.

Planning to test / review this PR later today / start of next week.

**Comment by reneaaron** [comment]
@GabrielSoga Great job with this PR! :+1: From a functional perspective everything seemed to work as expected, I've pushed some minor improvements / fixes to this branch here: 

https://github.com/getAlby/lightning-browser-extension/tree/budgets-new-ux
(for some reason I am not able to push to your branch, maybe you need to allow maintainers of getAlby to edit this PR?)

Will continue with the code review next week. Have a nice weekend! :sunglasses:

**Comment by fczuardi** [comment]
> @fczuardi @GabrielSoga Thanks for this PR! 👍 Could you please have a look at the unit tests? Some of them seem to fail currently.
> 
> Planning to test / review this PR later today / start of next week.

Unit tests are updated.

**Comment by fczuardi** [comment]
> Thanks for the PR! I can't test it without a build though

Builds are good now :)

**Comment by stackingsaunter** [comment]
Tested, looks really nice, great job @GabrielSoga !!!

Thanks you!

**Review by stackingsaunter — APPROVED** [review]
tAck

**Comment by fczuardi** [comment]
@reneaaron can we do something to help this branch? what are the next steps?

File: src/i18n/locales/en/translation.json [hunk 1]
```diff
@@ -1108,13 +1114,15 @@
     },
     "publishers_table": {
       "payments": "payments",
-      "used": "used"
+      "budget": "budget",
+      "total": "total"
     },
     "badge": {
       "label": {
         "active": "ACTIVE",
```
Comment (reneaaron) [inline src/i18n/locales/en/translation.json:1121]: Can we delete active here? It has been replaced with "budget" I think.

**Review by reneaaron — APPROVED** [review]
Thanks very much again for this PR, great work guys! :100: 

I've added some minor fixes (responsive layout, color usage, created a Modal component to avoid code reuse, etc) and left a few comments. 

Apart from the "division by zero" this looks good to merge from my side. 

tACK

**Comment by fczuardi** [comment]
nice cleanup! we like it :)

**Pull request merged**